src/index.html

```diff
@@ -7,8 +7,27 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a> 
+      <nav class="nav">
+        <ul class="menu">
+          <li class="list is-active">Apple</li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link  is-active" href="#apple">Apple</a>
          </li>
```
I think it is main reason why some tests failed
each list item should contain link
consider better naming of classes